app/views/admin/PageClanEdit.vue

```diff
@@ -80,6 +74,11 @@ export default {
       <p><a :href="`/league/${clan.slug}`">Esports page</a></p>
       <p><a :href="`/clan/${clan.slug}`">Legacy clan page</a></p>
 
+      <h2>Clan Stats</h2>
+      <ul>
+        <li>{{(clan.members || []).length}} members</li>
```

Comment:
This is another internal admin page. Added the number of members which is frequently asked for.